internal/functions/serve/serve.go

```diff
@@ -101,32 +89,26 @@ func Run(slug string, envFilePath string, verifyJWT bool) error {
 	}
 
 	// 4. Start Function.
-	{
-		fmt.Println("Starting " + utils.Bold("supabase/functions/"+slug))
-		out, err := utils.DockerExec(ctx, utils.DenoRelayId, []string{
-			"deno", "cache", "/home/deno/functions/" + slug + "/index.ts",
-		})
-		if err != nil {
-			return err
-		}
-		if _, err := stdcopy.StdCopy(io.Discard, io.Discard, out); err != nil {
-			return err
-		}
+	localFuncDir := filepath.Join(utils.FunctionsDir, slug)
+	relayFuncPath := filepath.Join(relayFuncDir, slug, "index.ts")
```

Comment:
spitballing: wouldn't this use the `\` separator on Windows instead of `/`?

Reply:
good catch. changed to unix path separator